src/scripts/main.js

```diff
@@ -1,3 +1,56 @@
 'use strict';
 
-// write code here
+const appendRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const appendColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+
+const table = document.querySelector('table');
+
+const MAX_ROWS_COUNT = 10;
+const MAX_COLUMNS_COUNT = 10;
+const MIN_ROWS_COUNT = 2;
+const MIN_COLUMNS_COUNT = 2;
```

Comment:
Don't need to duplicate variables with the same values
```suggestion
const MAX_COUNT = 10;
const MIN_COUNT = 2;
```